tests/testthat/_snaps/conditions.md

```diff
@@ -124,6 +124,7 @@
       x These names are duplicated:
         * "x" at locations 1, 2, and 3.
         * "y" at locations 4 and 5.
+      i Use argument `"check_unique"` to specify repair strategy.
```

Comment:
@lionel-, This doesn't look correct, right?

Reply:
oh I see what you mean now. Hmmmm